src/adapter/threads.ts

```diff
@@ -845,11 +846,14 @@ export class Thread implements IVariableStoreDelegate {
         };
       default:
         if (event.hitBreakpoints && event.hitBreakpoints.length) {
+          const isStopOnEntry =
+            event.hitBreakpoints.length === 1 &&
+            this._delegate.entryBreakpointId === event.hitBreakpoints[0];
```

Comment:
```suggestion
          const isStopOnEntry =
           this._delegate.entryBreakpointId &&
           event.hitBreakpoints.includes(this._delegate.entryBreakpointId);
```

in case we stop there for some other random reason

Reply:
My thinking was that if the user sets a breakpoint on this same place, then we don't want VS to auto-resume, so we want the breakpoint to take precedence and say "breakpoint" as a reason instead of "entry"

Reply:
ah, right, makes sense